k8s-daemonset/k8s/namerd.yml

```diff
@@ -1,11 +1,16 @@
 ---
-kind: ThirdPartyResource
-apiVersion: extensions/v1beta1
+kind: CustomResourceDefinition
```

Comment:
we're updating from TPR -> CRD in a larger doc update (#203). would you mind backing this particular change out of your PR?